content/product-engineering/planning.md

```diff
@@ -0,0 +1,183 @@
+# Product and engineering planning
+
+- [Product and engineering planning](#product-and-engineering-planning)
```

Comment:
Do we need both a manually written table of contents in addition to the automatically generated one to the right?

Reply:
I found this one helpful (and it is auto-generated by me via extension), but I will remove it. The index to the right in the handbook is missing two things IMO:
1. I think it should include H2s under H1s with indent.
1. Admittedly personal preference, but I think outlines should be on the left. On the right feels too hidden.

Issue filed here: https://github.com/sourcegraph/handbook/issues/901

Reply:
I actually like them on the left too, and I don't like the way it hides h2s today. At the same time it does feel strange to have two copies of the table of contents side by side when you go to the page. Maybe there's an easy fix for the official table of contents.